core/src/replicator.rs

```diff
@@ -457,6 +466,7 @@ impl Replicator {
             self.hash,
             self.slot,
             Signature::new(&self.signature.to_bytes()),
+            proof_index,
```

Comment:
What is proof_index for?

Reply:
It can be used by the _new_ validators to figure out how many random numbers to generate before they can start validating replicator proofs (i.e. catch up to validator's random number)

Reply:
Ok. I see, it could allow for missing transactions.